Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListRowWithValidationFailureComparerTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Moq;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Validation;
+using Remotion.ObjectBinding.Web.UI.Controls;
+using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls
+{
+  [TestFixture]
+  public class BocListRowWithValidationFailureComparerTest
+  {
+    private BocListRowWithValidationFailureComparer _comparer;
+    private IBocListValidationFailureRepository _validationFailureRepository;
+
+    private IBusinessObject _zeroErrorBusinessObject;
+    private IBusinessObject _singleErrorBusinessObject;
+    private IBusinessObject _doubleErrorBusinessObject;
+
+    private readonly BocListRow _nullRow = null;
+    private BocListRow _zeroErrorRow;
+    private BocListRow _singleErrorRow;
+    private BocListRow _doubleErrorRow;
+
+    [SetUp]
+    public void Setup ()
+    {
+      _validationFailureRepository = new BocListValidationFailureRepository();
+      _comparer = new BocListRowWithValidationFailureComparer(_validationFailureRepository);
+
+      _singleErrorBusinessObject = new Mock<IBusinessObject>().Object;
+      _zeroErrorBusinessObject = new Mock<IBusinessObject>().Object;
+      _doubleErrorBusinessObject = new Mock<IBusinessObject>().Object;
+
+      _zeroErrorRow = new BocListRow(0, _zeroErrorBusinessObject);
+      _singleErrorRow = new BocListRow(1, _singleErrorBusinessObject);
+      _doubleErrorRow = new BocListRow(2, _doubleErrorBusinessObject);
```

Comment:
```suggestion
      _rowWithoutErrors = new BocListRow(0, _zeroErrorBusinessObject);
      _rowWithOneError = new BocListRow(1, _singleErrorBusinessObject);
      _rowWithMutipleErrors = new BocListRow(2, _doubleErrorBusinessObject);
```
`single` / `double` has a different semantic. Particular `double` infers two of the same, not just two items. And our test focuses on the actual number not making a difference, so we don't even need to go with `two` but can use the more general `multilple`.